lib/modules/hosts/znipe.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+
+import { Host } from '../../core/host';
+
+export default new Host('twitchclips', {
```

Comment:
Can you update "twitchclips" to "znipetv" please :)

Reply:
hehe yeah maybe a good idea, thank you 😄